app/src/main/java/com/github/h3lp3rs/h3lp/LocalEmergencyCaller.kt

```diff
@@ -0,0 +1,105 @@
+package com.github.h3lp3rs.h3lp
+
+import android.content.Context
+import android.location.Geocoder
+import com.opencsv.CSVReader
+import java.io.BufferedReader
+import java.io.InputStreamReader
+
+/**
+ * Class containing methods enabling the Call Emergency Services button to make the number called
+ * match to the emergency services number in the user's current location
+ */
+class LocalEmergencyCaller {
```

Comment:
Why bother with a class if it only has a companion object? Might as well simply define LocalEmergencyCaller  as an object directly.

Reply:
Great remark, it should obviously be an object.